### PR TITLE
Updated the query and labels in the view and explore

### PR DESCRIPTION
**Pull request by sri-byte**
1) Impact: Updated the query and cleaned up the view. Added labels along with the type of data in the field, to make it easier to filter. 
2) Testing: Changes have been tested locally and work as expected.

### CODE REVIEW

File: models/data_warehouse.model.lkml [hunk 1]
```diff
@@ -3957,7 +3957,7 @@ explore: performance_events {
 
 explore: 1day_retention_rate {
   from: 1day_retention_rate
-  label: " Growth Scorecard 1-day Retention Rate"
+  label: "1-day Retention Rate"
```
Comment (jasonblais) [inline models/data_warehouse.model.lkml:3960]:
```suggestion
  label: "Day 1 Retention Rate"
```

The technical term is Day 1 Retention. 1-day retention measures it slightly differently. I'm sorry I just realized this, would it be too much effort to change the explore name to `day1_retention_rate` as well?

File: derived_views/growth_scorecard/1day_retention_rate.view.lkml [hunk 2]
```diff
@@ -1,26 +1,23 @@
...
-      WHERE (NOT (CASE WHEN server_fact.installation_id IS NOT NULL THEN TRUE ELSE FALSE END ) OR (CASE WHEN server_fact.installation_id IS NOT NULL THEN TRUE ELSE FALSE END ) IS NULL) AND (CASE WHEN server_fact.first_server_edition = 'true' THEN 'E0' WHEN server_fact.first_server_edition = 'false' THEN 'TE' ELSE NULL END) = 'E0' AND (server_fact.FIRST_ACTIVE_DATE) >= (TO_TIMESTAMP('2020-02-02')) AND (server_fact.FIRST_ACTIVE_DATE) > ((DATEADD('month', 0, DATE_TRUNC('month', CURRENT_DATE())))) AND excludable_servers.REASON IS NULL
-      GROUP BY
-      server_fact.FIRST_ACTIVE_DATE,
-          1) ww
-      ) bb WHERE z__pivot_col_rank <= 16384
-      ) aa
-      ) xx
-      ) zz
-       WHERE (z__pivot_col_rank <= 50 OR z__is_highest_ranked_cell = 1) AND (z___pivot_row_rank <= 500 OR z__pivot_col_ordering = 1) ORDER BY z___pivot_row_rank
+    sql: SELECT first_active_date
```
Comment (jasonblais) [inline derived_views/growth_scorecard/1day_retention_rate.view.lkml:3]: Let's add a brief description on what this query does, so we have it for future reference.

**Review by jasonblais — CHANGES_REQUESTED** [review]

File: derived_views/growth_scorecard/1day_retention_rate.view.lkml (hunk 2 above)
Comment (sri-byte) [reply derived_views/growth_scorecard/1day_retention_rate.view.lkml]: Added description as comments

File: models/data_warehouse.model.lkml (hunk 1 above)
Comment (sri-byte) [reply models/data_warehouse.model.lkml]: Updated the labels and titles. Keeping dbt stuff as is, for consistency.

**Review by jasonblais — APPROVED** [review]

**Pull request merged**